fgpyo/util/metric.py

```diff
@@ -75,15 +86,15 @@
 
 .. code-block:: python
 
-   >>> @attr.s(auto_attribs=True, frozen=True)
+   >>> @dataclasses.dataclass(frozen=True)
    ... class Name:
    ...     first: str
    ...     last: str
    ...     @classmethod
    ...     def parse(cls, value: str) -> "Name":
    ...          fields = value.split(" ")
    ...          return Name(first=fields[0], last=fields[1])
-   >>> @attr.s(auto_attribs=True, frozen=True)
+   >>> @dataclasses.dataclass(frozen=True)
    ... class Person(Metric["Person"]):
    ...     name: Name
    ...     age: int
```

Comment:
Is it worth it to have docs showing what happens if you mix dataclasses and attr?

Reply:
It turns out you literally can't mix them. The constructors don't work if you try. I added a short blurb in the docs to mention that, but no one will be able to make such an object and presumably will discover this in debugging.